main.py

```diff
@@ -0,0 +1,182 @@
+                })
+            data.append({"month": date.strftime('%B'), "records": records})
+    return data
+
+
+def get_formatted_date(date):
+    year, month, day = date.split('-')
+    return datetime.date(int(year), int(month), int(day)).strftime('%B %d')
+
+
+def print_yearly_data(data):
+    highest = {"temperature": 0, "date": ''}
+    lowest = {"temperature": 1000, "date": ''}
+    humidest = {"temperature": 0, "date": ''}
+
+    for month_data in data:
+        for record in month_data['records']:
+            if record['highest_temperature'].isnumeric() \
+                    and int(record['highest_temperature']) > highest['temperature']:
+                highest['temperature'] = int(record['highest_temperature'])
+                highest['date'] = get_formatted_date(record['date'])
+            if record['lowest_temperature'].isnumeric() and int(record['lowest_temperature']) < lowest['temperature']:
+                lowest["temperature"] = int(record['lowest_temperature'])
+                lowest['date'] = get_formatted_date(record['date'])
+            if record['max_humidity'].isnumeric() and int(record['max_humidity']) > humidest['temperature']:
+                humidest['temperature'] = int(record['max_humidity'])
+                humidest['date'] = get_formatted_date(record['date'])
+    print(f"Highest: {highest['temperature']}C on {highest['date']}")
+    print(f"Lowest: {lowest['temperature']}C on {lowest['date']}")
+    print(f"Humidity: {humidest['temperature']}% on {humidest['date']}")
+
+
+def check_month_in_command(command_data):
+    data = command_data.split('/')
+    return False if len(data) == 1 else True
+
+
+def get_month_data_from_data(data, month):
+    return next((item for item in data if item["month"] == month), None)
+
+
+def print_monthly_average_data(data, command_data):
+    highest_temp_data = {
+        "sum": 0,
+        "count": 0,
+    }
+    lowest_temp_data = {
+        "sum": 0,
+        "count": 0,
+    }
+    humidity_temp_data = {
+        "sum": 0,
+        "count": 0
+    }
+    for record in data['records']:
+        if record['highest_temperature'] != '':
+            highest_temp_data['sum'] += int(record['highest_temperature'])
+            highest_temp_data["count"] += 1
+        if record['lowest_temperature'] != '':
+            lowest_temp_data["sum"] += int(record['lowest_temperature'])
+            lowest_temp_data["count"] += 1
+        if record['mean_humidity'] != '':
+            humidity_temp_data["sum"] += int(record['mean_humidity'])
+            humidity_temp_data["count"] += 1
+
+    highest_average = highest_temp_data["sum"] // highest_temp_data["count"]
+    lowest_average = lowest_temp_data["sum"] // lowest_temp_data["count"]
+    average_humidity = humidity_temp_data["sum"] // humidity_temp_data["count"]
+
+    print(f'Highest Average: {highest_average}')
+    print(f'Lowest Average: {lowest_average}')
+    print(f'Average Mean Humidity: {average_humidity}')
```

Comment:
Make sure that we use double quotes through out our code. Try to remain consistent in your code style.